utilities/virt.py

```
@@ -297,6 +299,7 @@ def __init__(
         vm_affinity=None,
         annotations=None,
         label=None,
+        exclude_from_descheduler: bool | None = None,
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)